kuttl/kuttl-test-helm.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+testDirs:
+- ./kuttl/tests/helm
+parallel: 1
+timeout: 500
+startKIND: false
+artifactsDir: kuttl-artifacts
+kindNodeCache: false
+commands:
+  - command: bash -c "helm upgrade --install splunk-operator helm-chart/splunk-operator --namespace splunk-operator --create-namespace"
```

Comment:
where will the install happens, are we expecting helm charts are already installed ?

Reply:
This command would install the splunk-operator if not present. It would do nothing if its already there.

I added it here as it would help when running the tests locally